tests/tests/cpld_fv3_ccpp_mom6_cice_cmeps_1stepcold_atm_flux

```diff
@@ -0,0 +1,91 @@
+#
+#  CPLD_FV3_CCPP_MOM6_CICE_CMEPS_1STEPCOLD_ATM_FLUX test
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system - 1 step cold start"
+
+export CNTL_DIR="RT-Baselines_1stepcold_ccpp_cmeps"
+export CNTLMED_DIR="RT-Baselines_1stepcold_ccpp_cmeps"
```

Comment:
I think can also be an empty string since there is no control mediator directory.